spec/hello_world_spec.rb

```diff
@@ -0,0 +1,10 @@
+def greeting
+  'Hello world!'
+end
+
+describe 'Greeting' do
+  it 'returns message' do
+    message = greeting()
```

Comment:
You don't need parenthesis to call methods that don't take parameters in ruby.

```suggestion
	    message = greeting
```